build/Build.cs

```diff
@@ -189,6 +189,33 @@ class Build : NukeBuild
         });
     });
 
+    Target PackMergedClientNugetForMacOs => _ => _
+     .OnlyWhenStatic(() => IsLocalBuild)
```

Comment:
Is the indenting a bit funny here or is it just github displaying it?

I'd expect this, in keeping with the rest of the file

```csharp
    Target PackMergedClientNugetForMacOs => _ => _
         .OnlyWhenStatic(() => IsLocalBuild) // note tabbed in by 4 spaces here
         .OnlyWhenStatic(() => OperatingSystem.IsMacOS())
```

Reply:
Yeah I think my Rider is not doing this right. Let me fix that up.

Reply:
I think that's fixed now.